.env

```diff
@@ -111,6 +111,8 @@ PARQUET_EXPORT_DATASET=
 PARQUET_EXPORT_HF_TOKEN=
 PARQUET_EXPORT_SECRET=
 
+ADMIN_API_SECRET=# secret to admin API calls, like computing usage stats
```

Comment:
`PARQUET_EXPORT_SECRET` could be folded into `ADMIN_API_SECRET`

Reply:
yes agree with that suggestion^